librarian-puppet.gemspec

```
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   automatically pulling in modules from the forge and git repositories with
   a single command.'
 
-  s.required_ruby_version = '>= 2.4.0', '< 4'
+  s.required_ruby_version = '>= 2.6.0', '< 4'
```

If we're dropping old versions, I'd prefer to move it straight to 2.7+
```suggestion
  s.required_ruby_version = '>= 2.7', '< 4'
```

---

I think this means removing Puppet 5 from the test matrix - currently that's excluded from tests on 2.7 and 3.0 and it's EOL anyway so it doesn't feel unreasonable to drop support for that either, given a major version bump.

I'll add the necessary changes.

---

Puppet 5 used to be shipped with Puppet 2.4 anyway. It happened to run on Ruby 2.6, but that was not what the AIO version was. And as you say, it's EOL. Even Puppet 6 is going EOL soon.